### PR TITLE
fix reset password link

### PR DESCRIPTION
**Pull request by j0flintking02**
### Description 
this is to fix the bug that was in the email link that was meant to allow the user to rest the password

### CODE REVIEW

**Comment by nakimera** [comment]
LGTM

**Comment by balaakagordon** [comment]
Great work! However, the GET request returns a message with a typo: "this token is valid for asmall amount of time". Please fix this. Otherwise, LGTM

**Review by balaakagordon — APPROVED** [review]
Thanks for making the changes. LGTM

File: authors/apps/authentication/views.py [hunk 1]
```diff
@@ -115,12 +115,27 @@ def post(self, request):
...
+
+        return Response(serializer.data,
+                        status=status.HTTP_200_OK)
 
     def retrieve(self, request, token, *args, **kwargs):
         """this allows one to retrive the token from the end point
         """
-        msg = {'message': 'a link has been sent to your email', 'token': token}
+        msg = {
+            'message': 'this token is valid for a'
```
Comment (lym) [inline authors/apps/authentication/views.py]: Replace these two lines with
```
'message': 'this token is valid for a small amount of time',
'token': token
```
It's much easier to read.

File: authors/apps/authentication/views.py [hunk 2]
```diff
@@ -115,12 +115,27 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
 
-        return Response(serializer.data, status=status.HTTP_200_OK)
+        recipient = serializer.data['email']
+        subject = 'Activate Authors Haven Account'
+        host = 'http://' + request.get_host()
+        url = '/api/password-reset/'
+        token = serializer.data['token']
+        content = "Follow this link to Reset your"
```
Comment (lym) [inline authors/apps/authentication/views.py]: Replace these two lines with

```
content = "Follow this link to Reset your account password {}{}{}/".format(
    host, url, token
)
```
It's much easier to read.

Comment (j0flintking02) [reply authors/apps/authentication/views.py]: alright

**Pull request merged**